notifications_utils/template.py

```diff
@@ -379,7 +380,6 @@ def preheader(self):
             .then(strip_unsupported_characters)
             .then(add_trailing_newline)
             .then(notify_email_preheader_markdown)
-            .then(remove_language_divs)
```

Comment:
ditto?

Reply:
Thanks for the heads up, that was a mistake.